charts/opentelemetry-operator/templates/deployment.yaml

```
@@ -37,6 +37,9 @@ spec:
     spec:
       automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
       hostNetwork: {{ .Values.hostNetwork }}
+      {{- if and (semverCompare ">=1.33-0" .Capabilities.KubeVersion.Version) (kindIs "bool" .Values.hostUsers) }}
```

Using `.Values.hostUsers` is acceptable.


```suggestion
      {{- if and (semverCompare ">=1.33-0" .Capabilities.KubeVersion.Version) .Values.hostUsers }}
```